x-pack/plugins/data_visualizer/public/application/index_data_visualizer/constants/random_sampler.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const RANDOM_SAMPLER_PROBABILITIES = [
+  0.5, 0.25, 0.1, 0.05, 0.025, 0.01, 0.005, 0.0025, 0.001, 0.0005, 0.00025, 0.0001, 0.00005,
+  0.00001,
+]
+  .reverse()
+  .map((n) => n * 100);
+
+export const RANDOM_SAMPLER_STEP = 0.00001 * 100;
+
+export const RANDOM_SAMPLER_OPTION = {
+  ON_AUTOMATIC: 'on_automatic',
+  ON_MANUAL: 'on_manual',
+  OFF: 'off',
+} as const;
+
+export type RandomSamplerOption = typeof RANDOM_SAMPLER_OPTION[keyof typeof RANDOM_SAMPLER_OPTION];
+
+export const RANDOM_SAMPLER_SELECT_OPTIONS: Array<{ value: RandomSamplerOption; text: string }> = [
+  {
+    value: RANDOM_SAMPLER_OPTION.ON_AUTOMATIC,
+    text: i18n.translate('xpack.dataVisualizer.randomSamplerPreference.onAutomaticLabel', {
+      defaultMessage: 'On (automatic use best %)',
```

Comment:
The text for the 'On' options needs some tweaking I think. `On (automatic configuration)` `On (manual configuration)` ? @lcawl any suggestions?

This is all about balancing speed against accuracy. We want to encourage the user to leave it as 'automatic'.

Reply:
Yes, I think if we can avoid using "%" in the label (and thus avoid having to explain what that percent actually means), that'd be simpler. Maybe even as simple as "On (automatic)" and "On (manual)"